polylogue/cli/imports.py

```
@@ -226,13 +239,29 @@ def run_import_codex(args: argparse.Namespace, env: CommandEnv) -> None:
     export_path = Path(args.export_path)
+
+    # Validate export file exists
+    if not export_path.exists():
+        json_mode = getattr(args, "json", False)
+        if json_mode:
+            raise JSONModeError(
+                "file_not_found", f"Export file not found: {export_path}", path=str(export_path)
+            )
+        else:
+            console.print(f"[red]Error: Export file not found: {export_path}")
+            console.print(f"[dim]Check that the path is correct and the file exists.")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)